cves/kernel/CVE-2021-38160.yml

```diff
@@ -221,7 +222,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: drivers
```

Comment:
File system (fs) is also present, since copying and buffers apply directly to file structure.

Reply:
I think this is primarily a drivers issue, as that is the subsystem the bug is located in, and tagging every subsystem that uses buffers would be more information than is useful